docs/source/tutorials/hospital_admissions_model.qmd

```diff
@@ -166,11 +168,11 @@ latent_inf = latent.Infections()
 I0 = InfectionInitializationProcess(
     "I0_initialization",
     metaclass.DistributionalRV(
-        dist=dist.LogNormal(loc=jnp.log(100), scale=0.5), name="I0"
+        dist=dist.LogNormal(loc=jnp.log(100), scale=jnp.log(1.75)), name="I0"
     ),
     InitializeInfectionsExponentialGrowth(
         gen_int_array.size,
-        deterministic.DeterministicVariable(0.5, name="rate"),
+        deterministic.DeterministicVariable(0.05, name="rate"),
```

Comment:
Why the change?

Reply:
Assume it's just to make the fit better.

Reply:
More realistic given the actual data we're fitting to we're going to fix a value.